tasks/vscode.py

```diff
@@ -73,74 +46,15 @@ def setup_devcontainer(
     """
     Generate or Modify devcontainer settings file for this project.
     """
-    flavor = AgentFlavor[flavor]
-    if target not in build_tags[flavor]:
-        print("Must choose a valid target.  Valid targets are: \n")
-        print(f'{", ".join(build_tags[flavor].keys())} \n')
-        return
-
-    build_include = (
-        get_default_build_tags(build=target, arch=arch, flavor=flavor)
-        if build_include is None
-        else filter_incompatible_tags(build_include.split(","), arch=arch)
+    from tasks.devcontainer import setup
+
+    print('This command is deprecated, please use `devcontainer.setup` instead')
```

Comment:
Maybe we should output this to stderr + in red. We can add also `Running devcontainer.setup task` such that we know that it worked

Reply:
Good idea, added!